signup-front/package.json

```diff
@@ -51,12 +51,12 @@
     "react-table": "^7.5.2",
     "react-table-6": "^6.11.0",
     "recharts": "^2.0.9",
-    "serve": "^11.3.2",
```

Comment:
are the updates in package.lock for these changes ? or are they due to a late migration to node 16 ?

Reply:
both :/